packages/components/src/components/dropdown/Dropdown.tsx

```diff
@@ -290,7 +292,8 @@ export class Dropdown<T = LabelValue> extends React.Component<
     menuPortalStyles: {},
     menuShouldBlockScroll: false,
     menuShouldScrollIntoView: true,
-    size: 'medium'
+    size: 'medium',
+    bindLabel: (option) => option.label,
```

Comment:
Why we need to have a defaultValue if it's not provided?

Reply:
@symphony-annaLlorens If we use an option structure which is different from `{ label: '', value: '' }`, we have to provide a way to render the option text, through `bindLabel` (which actually corresponds to the `getOptionLabel` prop of the form-react-select components - along with the `getOptionValue`).

```js
<Dropdown 
    options={[
        { uid: '1', test: 'option 1'},
        { uid: '2', test: 'option 2'},
    ]} 
    bindLabel={(option) => option.text}
    bindValue={(option) => option.uid} 
/>
```

If we don't provide the `bindLabel` it will try to render the default `option.label`, and we will end up with the result of first video (see PR description).